configs/vue3.js

```diff
@@ -26,5 +26,7 @@ module.exports = {
       },
     ],
     'vue/html-closing-bracket-spacing': 'error',
+    'vue/no-empty-component-block': 'error',
+    'vue/html-button-has-type': 'warn',
```

Comment:
Is het mogelijk om iets als "vue-generic.js" te maken waarin regels die op vue 2 & 3 van toepassing zijn neer te zetten?